packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -275,19 +287,62 @@ const RewardsTile = ({ className }: RewardsTileProps) => {
 
   const wm = useWithMobileStyle(styles.mobile)
 
+  const totalClaimableAmount = Object.values(optimisticUserChallenges).reduce(
+    (sum, challenge) => sum + challenge.claimableAmount,
+    0
+  )
+  const pendingChallengeSchedule = usePendingChallengeSchedule()
+
+  const onClickClaimAllRewards = () => {
+    setVisibility('ClaimAllRewards')(true)
+  }
+  const pendingAmount = pendingChallengeSchedule.claimableAmount
+
   return (
     <Tile className={wm(styles.rewardsTile, className)}>
       <span className={wm(styles.title)}>{messages.title}</span>
       <div className={wm(styles.subtitle)}>
         <span>{messages.description1}</span>
       </div>
-      <div className={styles.rewardsContainer}>
-        {userChallengesLoading && !haveChallengesLoaded ? (
-          <LoadingSpinner className={wm(styles.loadingRewardsTile)} />
-        ) : (
-          rewardsTiles
-        )}
-      </div>
+      {userChallengesLoading && !haveChallengesLoaded ? (
+        <LoadingSpinner className={wm(styles.loadingRewardsTile)} />
+      ) : (
+        <>
+          {isRewardsCooldownEnabled ? (
+            <div className={wm(styles.claimAllContainer)}>
+              <IconTokenGold
+                height={48}
+                width={48}
```

Comment:
checked w design and confirmed we want 48x48, this is a special icon so the utility icon sizing doesn't apply. also we should probably update the icons in harmony since they're currently 50x50 https://www.figma.com/file/DjbXq9opkpWHpZGaKlGGEt/%E2%9A%9B%EF%B8%8F-Icon-Library?type=design&node-id=2432-6195&mode=design&t=2ipEIsON72rhC0Wg-0